src/powershell-unix/powershell-unix.csproj

```diff
@@ -45,6 +45,11 @@
     <DefineConstants>$(DefineConstants);UNIX</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)' == 'LinuxDebug' ">
+    <DebugType>portable</DebugType>
```

Comment:
They are generated. However, when specifying `-Crossgen`, those .pdb files will be removed during `Start-Crossgen`.